src/libponyc/CMakeLists.txt

```diff
@@ -139,7 +152,7 @@ else()
         COMMAND echo "addlib $<TARGET_FILE:libponyc>" >> standalone.mri
         COMMAND echo "save" >> standalone.mri
         COMMAND echo "end" >> standalone.mri
-        COMMAND ar -M < standalone.mri
+        COMMAND ${CMAKE_AR} -M < standalone.mri
```

Comment:
This is also changing something related to DragonFly, has this been tested on a dragonfly system?